float64 v
float64 yaw_rate
